all_protos/cometbft/store/v1/types.proto

```
@@ -0,0 +1,10 @@
+syntax = "proto3";
+package cometbft.store.v1;
```

Buf (1.54.0)

Files with package "cometbft.store.v1" must be within a directory "cometbft/store/v1" relative to root but were in directory "all_protos/cometbft/store/v1".

(PACKAGE_DIRECTORY_MATCH)